src/cmds/validate.ts

```diff
@@ -1,5 +1,5 @@
-import type { CommandOptions } from '../lib/baseCommand';
 import type { Options } from './openapi/validate';
+import type { CommandOptions } from '../lib/baseCommand';
```

Comment:
weird, this reorder shouldn't have happened.

Reply:
lol yikes... could this be another bug with that import plugin? i can hold off

Reply:
Looks like it's just `type` imports that are this way. I'll update that ticket I had

Reply:
Looks like they're already on it. Feel free to pull this bump in anyways and we can change this back when they fix it.

Reply:
eh I'll just convert this to a draft in the meantime, I'd rather keep the diff minimal